google/cloud/bigtable/client.py

```diff
@@ -440,7 +494,7 @@ async def read_rows_sharded(
         per_row_timeout: int | float | None = 10,
         idle_timeout: int | float | None = 300,
```

Comment:
Can we remove this from the API? I don't think we want customers to configure it.